### ISSUE TITLE
Is this library still active?

### ISSUE THREAD

**Issue by andrewlsimplisafe** [rnosov/react-reveal#69]
Asking because the last deploy was about a year ago and many issues on this board have went unaddressed.

**Comment by rnosov** [comment]
Yes it's still active. I've been launching some big projects at my day job plus dealing with some health issues so I couldn't put much time into this project this year. The project is due for a major rewrite that I need spend some time on but as it won't pay my medical bills I have to prioritise paying work at the moment. Hopefully, the problems I'm facing should be resolved by around October timeframe so version 2.0 might be coming around this time.

**Comment by le-du6** [comment]
Good news for your health.
Thank you very much for taking over the maintenance of this fabulous component. It is simple, direct and works perfectly.

**Comment by sshanzel** [comment]
If you like, we can help you do the refactoring you're planning. Just pitch it out and i'll be sure to help! I think we can utilize GitHub's features to coordinate with each other

**Comment by FernandoChu** [comment]
Count me int!

**Comment by venelinn** [comment]
I hope everyone's fine !
Any updates on version 2.0 of this amazing library?
